lib/tasks/maintenance.rake

```diff
@@ -43,4 +43,38 @@ namespace :maintenance do
       from_stage.destroy! if delete_empty && from_stage.deploy_groups.empty?
     end
   end
+
+  # FROM and TO handles multiple deploy groups to not mess with projects that have each deploy group in a single stage.
+  desc "delete deploy groups across stages for all projects. OWNER= DELETE=dg1,dg2 [DELETE_EMPTY=true]"
```

Comment:
we usually remove just 1 ... and even if we remove 2 I'd rather run it twice
```suggestion
  desc "delete deploy groups across stages for all projects. OWNER= DELETE=dg1 [DELETE_EMPTY=true]"
```

Reply:
yeah I can see how that's not a good idea `alias watch_the_world_burn="rake maintenance:delete_deploy_groups DELETE=pod17,pod18,pod19,pod20"` :D